programs/github

```diff
@@ -108,8 +108,11 @@ if action == subcommands.clone then
 	local tree = repo:tree(treeName)
 
 	-- download the files
-	print("Downloading:")
 	local totalSize = tree.size
+	if totalSize > fs.getFreeSpace(dest) then
+		return printError(("Repository is %s, but only %s are free on this computer. Aborting!"):format(sizeStr(totalSize), sizeStr(fs.getFreeSpace(dest)), with sizeStr(1) == ' byte(s)', sizeStr(2048) == '2.0 KiB'))
```

Comment:
??? Notice how [my old comment](https://github.com/eric-wieser/computercraft-github/pull/15#discussion_r85009489) had some things not in code blocks!